packages/story-editor/src/components/library/panes/shapes/shapePreview.js

```diff
@@ -123,6 +120,10 @@ function ShapePreview({ mask, isPreview, index }) {
     dataToEditorY: state.actions.dataToEditorY,
   }));
 
+  const [active, setActive] = useState(false);
+  const makeActive = useCallback(() => setActive(true), []);
+  const makeInactive = useCallback(() => setActive(false), []);
```

Comment:
I added it mainly because saw the same thing done elsewhere, for consistency. We have an open issue for generally establishing best practices for these: #10467. I can remove from here though.